tests/acceptance/expected-failures-API-on-OCIS-storage.md

```diff
@@ -214,11 +214,15 @@ cannot share a folder with create permission
 
 #### [download previews of other users file](https://github.com/owncloud/ocis/issues/2071)
 
-- [coreApiWebdavPreviews/previews.feature:95](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/coreApiWebdavPreviews/previews.feature#L95)
+- [coreApiWebdavPreviews/previews.feature:165](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/coreApiWebdavPreviews/previews.feature#L165)
+- [coreApiWebdavPreviews/previews.feature:166](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/coreApiWebdavPreviews/previews.feature#L166)
+- [coreApiWebdavPreviews/previews.feature:167](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/coreApiWebdavPreviews/previews.feature#L167)
```

Comment:
IMHO, Let's not add this to expected failure. The failure is with a response message which can be easily fixed. Let's get the fix done and refactor the message in the new PR.

Reply:
Yeah. Let dev team push a fix PR for https://github.com/owncloud/ocis/issues/2071

CC @ScharfViktor @micbar @2403905 
or any opinions?

Reply:
#2071 has stayed open for over 2 years. I will inform dev team today.

Reply:
This could be the chance to cleanup that issue :smile: